packages/doltlab/content/guides/features/README.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Features"
+---
+
+DoltLab has all the features of [DoltHub](https://www.dolthub.com). DoltHub is modeled after GitHub and thus supports, viewing and editing databases on the web, Pull Requests, Issues, Permissions, and many other features. These features should be easy to understand and self-documenting. 
+
+Some more esoteric features require specific documentation like:
```

Comment:
```suggestion
Some more esoteric features that require specific documentation, like:
```

Reply:
I like the , don't need the that will add.